compiler/x/i386/codegen/OMRTreeEvaluatorTable.hpp

```diff
@@ -413,7 +413,7 @@
    TR::TreeEvaluator::lternaryEvaluator,                               // TR::lternary
    TR::TreeEvaluator::badILOpEvaluator,                                // TR::bternary
    TR::TreeEvaluator::badILOpEvaluator,                                // TR::sternary
-   TR::TreeEvaluator::badILOpEvaluator,                                // TR::aternary
+   TR::TreeEvaluator::iternaryEvaluator,                               // TR::aternary
```

Comment:
Please update the copyright header on this file.

Reply:
Addressed.